### DISCUSSION TITLE
entry not showing up after db modified elsewhere

### DISCUSSION THREAD

**Discussion by solsticedhiver** [Kunzisoft/KeePassDX#1766] [General]
So I manage my password db mostly on desktop with _keepassxc_ on linux. The file is put in a _dropbox_ flder to share it with my phone where I use _keepassdx_.

Last time, I had a problem but it was because _dropbox_ created a backup of the db after I modifed it on the phone with _keepassdx_. May be because the file was still opened somehow on my desktop.

But now, I modified my db on my desktop, close _keepassxc_, and _keepassdx_ shows me the file has been modifed at 2024-02-11 00:13. that's ok.

However, I am **unable** to find the entry I have added to the db.
Either by searching for term of the name of the entry (mpd), or by scrolling all the entries after sorting them.

I am clueless at what is wrong, because everything seems ok, except that I can't find my entry...

**Comment by J-Jamet** [comment]
If KeePassDX tells you that the file has been modified, simply reload the database from its current URI using the "Reload data" button.
If the data is still not reloaded correctly, the Dropbox application has not done its job of managing file conflicts correctly.
https://github.com/Kunzisoft/KeePassDX/wiki/File-Manager-and-Sync#save-merge-and-reload

**Reply by solsticedhiver** [reply]
Never mind, I found the reload data thing. It's under a hamburger menu.

I don't know what is happening. I have to force Dropbox to save a copy of the db file, so that somehow, it is available updated to keepassxd
May be use the off-line feature ...

No wait. the db file was marked as off-line file in dropbox, hence the problem; I think I fixed it by removing the off-line tag on that file now!
